airbyte-bootloader/src/test/java/io/airbyte/bootloader/BootloaderAppTest.java

```diff
@@ -80,7 +80,7 @@ void testBootloaderAppBlankDb() throws Exception {
         mockedConfigs.getConfigDatabaseUrl())
             .getAndInitialize();
     val configsMigrator = new ConfigsDatabaseMigrator(configDatabase, this.getClass().getName());
-    assertEquals("0.35.28.001", configsMigrator.getLatestMigration().getVersion().getVersion());
+    assertEquals("0.35.32.001", configsMigrator.getLatestMigration().getVersion().getVersion());
```

Comment:
@davinchia why do i need to change this test every time there is a migration? seems very manual.

Reply:
automatically updating expected tests always felt somewhat wrong to me. I don't feel too strongly so I'm happy to add this if you think it's alright.